cmd/compute-domain-kubelet-plugin/driver.go

```diff
@@ -199,6 +202,9 @@ func (d *driver) UnprepareResourceClaims(ctx context.Context, claimRefs []kubele
 			if done {
 				results[claim.UID] = err
 				wg.Done()
+				if err != nil {
+					klog.V(0).Infof("Permanent error unpreparing devices for claim %v: %v", claim.UID, err)
```

Comment:
Can the V(0) be dropped?

Reply:
Also, can the log be done inside `nodeUnprepareResource` so as not to clutter things here?

Reply:
Yes, level 0 is implicit when doing `klog.Infof()`.